kinetic/distribution.yaml

```diff
@@ -10839,6 +10839,23 @@ repositories:
       url: https://github.com/stonier/rqt_wrapper.git
       version: devel
     status: developed
+  rr_openrover_basic:
+    doc:
+      type: git
+      url: https://github.com/RoverRobotics/rr_openrover_basic.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/RoverRobotics/rr_openrover_basic-release.git
+      version: 0.0.1-0
+    source:
+      test_pull_requests: true
```

Comment:
Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, either remove this line or provide the necessary access as described in the wiki page.

Reply:
Removed